src/OverlayTrigger.js

```diff
@@ -178,30 +175,24 @@ const OverlayTrigger = React.createClass({
     // create in render otherwise owner is lost...
     this._overlay = this.getOverlay();
 
-    if (this.props.trigger !== 'manual') {
-
-      props.onClick = createChainedFunction(trigger.props.onClick, this.props.onClick);
+    props.onClick = createChainedFunction(trigger.props.onClick, this.props.onClick);
 
-      if (isOneOf('click', this.props.trigger)) {
-        props.onClick = createChainedFunction(this.toggle, props.onClick);
-      }
-
-      if (isOneOf('hover', this.props.trigger)) {
-        warning(!(this.props.trigger === 'hover'),
-          '[react-bootstrap] Specifying only the `"hover"` trigger limits the visibilty of the overlay to just mouse users. ' +
-          'Consider also including the `"focus"` trigger so that touch and keyboard only users can see the overlay as well.');
+    if (isOneOf('click', this.props.trigger)) {
+      props.onClick = createChainedFunction(this.toggle, props.onClick);
+    }
 
-        props.onMouseOver = createChainedFunction(this.handleDelayedShow, this.props.onMouseOver);
-        props.onMouseOut = createChainedFunction(this.handleDelayedHide, this.props.onMouseOut);
-      }
+    if (isOneOf('hover', this.props.trigger)) {
+      warning(!(this.props.trigger === 'hover'),
+        '[react-bootstrap] Specifying only the `"hover"` trigger limits the visibilty of the overlay to just mouse users. ' +
+        'Consider also including the `"focus"` trigger so that touch and keyboard only users can see the overlay as well.');
```

Comment:
those who will use it only for `mouse`-users.. will be annoyed and then one of two:
1) they will complain about it on issue trackers, or
2) they will just add focus to suppress this warning

maybe better to move this warning into `props.trigger` doc-definition ?